src/app/layout.js

```diff
@@ -1,14 +1,9 @@
-import { Geist, Geist_Mono } from "next/font/google";
+import { Inter } from "next/font/google";
 import "./globals.css";
 import StoryblokProvider from "@/components/StoryblokProvider";
 
-const geistSans = Geist({
-  variable: "--font-geist-sans",
-  subsets: ["latin"],
-});
-
-const geistMono = Geist_Mono({
-  variable: "--font-geist-mono",
+const inter = Inter({
+  variable: "--font-inter",
```

Comment:
Is it worth adding a custom font? I think we can go with `system-ui` and make it even simpler, require less packages, etc.

Reply:
Probably worth discussing @manuelschroederdev